profiles/migrations/0025_populate_image_small.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+from uuid import uuid4
+
+from django.db import migrations, models
+from profiles.util import make_thumbnail
+
+
+def populate_image_small(apps, schema_editor):
+    """
+    Populate image_small with thumbnail of image if it exists
+    """
+    Profile = apps.get_model('profiles.Profile')
+    for profile in Profile.objects.all():
+        if profile.image and not profile.image_small:
+            thumbnail = make_thumbnail(profile.image.file)
+            profile.image_small.save("{}.jpg".format(uuid4().hex), thumbnail)
```

Comment:
Just in case the name gets used in a way where it would collide with the other image names. I don't know if it matters